src/app.ts

```diff
@@ -1,6 +1,72 @@
+type ResultData = {
+    id: string,
+    gender: string,
```

Comment:
Codacy has a fix for the issue: [Replace `··gender:·string,` with `gender:·string;`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
  gender: string;
```